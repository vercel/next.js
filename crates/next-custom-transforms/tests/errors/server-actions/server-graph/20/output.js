export async function fn() {
    'use server';
}
