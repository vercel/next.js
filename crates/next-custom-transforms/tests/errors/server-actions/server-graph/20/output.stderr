  x Conflicting directives "use server" and "use cache" found in the same function body. You cannot place both directives at the top of a function body. Please remove one of them.
  | 
   ,-[input.js:3:1]
 2 |   'use cache'
 3 |   'use server'
   :   ^^^^^^^^^^^^
 4 | }
   `----
